ansible/roles/tilemaker/templates/bicycle/process-openmaptiles.lua

```diff
@@ -363,6 +363,10 @@ function way_function(way)
 			SetBrunnelAttributes(way)
 			if ramp then way:AttributeNumeric("ramp",1) end
 
+			-- No bike roads
+			if has_falsy_tag(way, "bicycle") or has_thruthy_tag(way, "motorroad") then
+				way:Attribute("no_bike", 1)
```

Comment:
Shouldn't we turn this around and say `bike=0`?

Reply:
Well, I used "has" and "!has" rules on the style (independently of the value), so it would be a weird description to see a layer with the name "something-no_bike" and have a filter that says "has bike". Or we should add this key to every road and make the rules based by its value.
I'll do if you think it's better but I wanted to go with the easiest (less modification) solution.

Reply:
Ok, you convinced me.